check_modulemd.py

```diff
@@ -133,8 +133,12 @@ def test_description(self):
             "Checking for presence of description that is properly punctuated")
         if self.mmd.description and len(self.mmd.description) > 0:
             if not self.mmd.description.endswith('.'):
-                self.error("Description should end with a period: %s" %
-                           self.mmd.description)
+                if self.mmd.description.endswith('.\n'):
```

Comment:
@contyk any suggestion here? I do not even know if it is possible for it to have more than 1 newlines in the end. But in this case I would say it is an error.